### ISSUE TITLE
Running the pycaw tests modifies the system and breaks audio

### ISSUE THREAD

**Issue by reversefold** [AndreMiras/pycaw#31]
I ran the tests via tox to verify my recent pull request and now my audio is broken. Something was changed and one of my audio devices was no longer working properly. I had to disable it and then re-enable it in the Sound Playback tab for it to work again.

The pycaw tests should not be modifying system they are running on. Any write operations should be mocked out or should be limited to some kind of defined test interface.

**Comment by AndreMiras** [comment]
Agree, we should have all the unit tests mocked and then some specific integration tests (not ran by default) not mocked.
Thanks for the report

**Comment by AndreMiras** [comment]
Eventually fixed via 8c44cd0. We're now using mocks except for some read-only ones
